02week/bio/css/style.css

```diff
@@ -0,0 +1,23 @@
+body{background-color: #d7dfea;
```

Comment:
good use of comments.  Hit return after the { and put the first property on the next line.